### PR TITLE
[16.0][MIG] stock_picking_return_refund_option: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT42442

### CODE REVIEW

File: stock_picking_return_refund_option/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Picking Return Refund Option",
     "summary": "Update the refund options in pickings",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```
Comment (pedrobaeza) [inline stock_picking_return_refund_option/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by pedrobaeza — COMMENTED** [review]
And please put the setup changes separated in the pre-commit stuff commit (see the migration guide)

File: stock_picking_return_refund_option/__manifest__.py (hunk 1 above)
Comment (Yadier-Tecnativa) [reply stock_picking_return_refund_option/__manifest__.py]: Changed

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Code and functional :+1:

**Review by sergio-teruel — APPROVED** [review]

**Comment by sergio-teruel** [comment]
ping @rafaelbn

**Comment by sergio-teruel** [comment]
Done! 
@juangomezzurita

**Pull request merged**